js/main.js

```diff
@@ -0,0 +1,11 @@
+(function($) {
+  $(function() {
+
+    $('.my-btn').sideNav({
```

Comment:
`my-btn` is a weird name. Please use something more consistent and meaningful, like a `menu-btn` or something.